student/src/main/webapp/WEB-INF/web.xml

```diff
@@ -145,6 +146,28 @@
 		<filter-name>contextInitializerFilter</filter-name>
 		<url-pattern>/*</url-pattern>
 	</filter-mapping>
+
+	<!--
+		This filter limits concurrent requests to an arbitrary value, sacrificing clients in
+		the name of preventing the server from being overloaded and preserving throughput.
+	-->
+	<filter>
+		<filter-name>concurrentRequestLimitFilter</filter-name>
+		<filter-class>org.springframework.web.filter.DelegatingFilterProxy</filter-class>
+	</filter>
+	<filter-mapping>
+		<filter-name>concurrentRequestLimitFilter</filter-name>
+		<url-pattern>/Pages/API/*</url-pattern>
+		<url-pattern>/Pages/Pages/API/*</url-pattern>
+		<url-pattern>/API.axd/*</url-pattern>
+		<url-pattern>/TIS/API/*</url-pattern>
+		<url-pattern>/Pages/SpellEngine.axd/*</url-pattern>
+		<url-pattern>/ItemScoringRubric.axd/*</url-pattern>
+		<url-pattern>/ItemScoringCallback.axd/*</url-pattern>
+		<url-pattern>/browsersessionlaunch</url-pattern>
+		<url-pattern>*.xhtml</url-pattern>
+	</filter-mapping>
```

Comment:
This limits the concurrent request limiting filter to heavy-load endpoints. (Things served by the Apache Faces servlet and Spring DispatcherServlet.)  I don't want to limit on retrieving .css, .js, etc static resources.